ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_dev/ExplicitDisclosureIT.scala

```diff
@@ -448,23 +444,15 @@ final class ExplicitDisclosureIT extends LedgerTestSuite {
           testContext.disclosedContract.update(_.metadata.modify(_.clearCreatedAt))
         )
         .mustFail("using a disclosed contract with missing createdAt in contract metadata")
-
-      //      // TODO ED: Assert missing contract key hash when ledger side metadata validation is implemented
-      //      // Exercise a choice using an invalid disclosed contract (missing key hash in contract metadata for a contract that has a contract key associated)
-      //      errorMissingKeyHash <- testContext
-      //        .exerciseFetchDelegated(
-      //          testContext.disclosedContract.update(_.metadata.contractKeyHash.set(ByteString.EMPTY))
-      //        )
-      //        .mustFail(
-      //          "using a disclosed contract with missing key hash in contract metadata for a contract that has a contract key associated"
```

Comment:
Assertion not needed since ledger side validation does not need to care about contract_key_hash (only used in interpretation)